src/gromacs/nblib/bondtypes.cpp

```diff
@@ -240,9 +246,15 @@ auto MorseBondType::properties()
 
 bool operator==(const MorseBondType& a, const MorseBondType& b)
 {
+#if GMX_DOUBLE
```

Comment:
@zhmurov your comments mirror my own that I made a few weeks ago. It would appear that they were marked as resolved without actually being addressed (or perhaps there was some sort of system error that means that only I can see them?). @kanduri did you mark my comments as resolved or did they get lost somehow?

Reply:
I assume that the PR changed a lot and your comments were not showing in the compare view. I can see your comments above on this page, but interface is such that it is impossible to relate them to the code without clicking the link. Maybe, they were indeed resolved by someone.

Reply:
I didn't mark any comments as resolved yet. Github works in mysterious ways.

Reply:
When I see your comments, I can only see them in the context of the previous commit, not with my updated patches. This kinda makes sense I feel.

Reply:
When creating a Review, you can make your feedback more "blocking" by marking it as "Request Changes" instead of "Comment"

Reply:
@ejjordan 
TIL that comments and review comments are treated differently. If you make comments in review mode, that can be seen in all patches and one can resolve them. If you make a generic comment, then it is only when viewing that particular patch.

Reply:
One cannot request changes if they posted the MR I think.